test/inductor/test_torchinductor.py

```diff
@@ -3216,6 +3216,63 @@ def fn(x):
         )
         assertGeneratedKernelCountEqual(self, 0)
 
+    @config.patch(fallback_random=True)
```

Comment:
Would it make more sense to pass the `_random_samples` argument to `fractional_max_pool2d_with_indices` so we remove the random generation from the equation?